File: 260312/더 긴 문자열/longer-string.c
#include <stdio.h>
#include <string.h>

int main() {
    // Please write your code here.
    char word1[20];
    char word2[20];
    scanf("%s %s", word1, word2);

    int len1 = strlen(word1);
    int len2 = strlen(word2);

    if (len1 > len2) {
        printf("%s %d", word1, len1);
    }
    else if (len1 < len2) {
        printf("%s %d", word2, len2);
    }
    else printf("same");

    return 0;
}

File: 260312/더 긴 문자열/longer-string.py
words = input().split()

word1 = words[0]
word2 = words[1]

len1 = len(word1)
len2 = len(word2)

if len1 > len2:
    print(f"{word1} {len1}")
elif len1 < len2:
    print(f"{word2} {len2}")
else:
    print("same")